src/scripts/site.js

```diff
@@ -1317,7 +1294,7 @@ var AppInfo = {};
 
         if (!window.Promise || browser.web0s) {
             initialDependencies.push();
```

Comment:
What is this?

Reply:
I forgot to remove initialDependencies. The objective here was to remove initialDependencies from the require method. For webpack to resolve the require modules, it can't be passed as an array like initialDependencies. So i refactored the method so i could make the require explicitly.